tree/treeplayer/test/dataframe/dataframe_cache.cxx

```diff
@@ -163,3 +191,62 @@ TEST(Cache, NonCopiable)
    EXPECT_EQ(0, ret)
       << "The static assert was not triggered even if caching of columns of a non copiable type was requested";
 }
+
+auto treeName = "t";
+auto fileName = "fileName.root";
+
+TEST(Cache, TakeCarrays)
+{
+   {
+      TFile f(fileName, "RECREATE");
```

Comment:
if we recreate it everytime anyway, we should delete this file when we are done with it

Reply:
why?

Reply:
to leave the testing directory clean. I know there are other places where we don't do it, but wouldn't it be good practice?